react-toggle-switch-component/my-app/src/App.js

```diff
@@ -0,0 +1,17 @@
+import ToggleSwitch from './ToggleSwitch';
+import { useState } from 'react';
+
+const App = () => {
+  const [isToggled, setIsToggled] = useState(false);
+  return (
+    <div className="App">
+      <ToggleSwitch
+        rounded={true}
```

Comment:
If you don't provide a value to a prop, the default value is `true`, so you could just write this as `<ToggleSwitch rounded isToggled={isToggled} onToggle={...} />`